### DISCUSSION TITLE
コメントがついたことの通知が届かないことが稀にある

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#552] [要望・アイデア・不具合]
記事にコメントしたことに対してメンション付きでコメントが返ってきていたのですが、通知アイコンや通知一覧に表示されてませんでした。
https://qiita.com/ment_RE/items/7dd5c00ed5f83bf058fa#comment-0ead5075976058909e99
以前にも同様なことがありました。
コメントを返したりお礼を伝えたりできなくて失礼な結果になってしまうことがあるので、改善していただけたら嬉しいです。

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご報告ありがとうございます。一旦調査いたします🙏

もし可能であればで構わないのですが、複数事例があると調査がしやすいため
次に同様の現象が起きた際にもサポートフォーム宛てに該当の記事とコメントをご連絡いただけるとありがたいです🙏
